doc/reference/guidelines.rst

```diff
@@ -44,16 +51,36 @@ For instance, *sale* module introduces ``sale_order`` and
 ``<main_model>`` files will be named :file:`models/sale_order.py` and
 :file:`views/sale_order_views.py`.
 
-
 For *data*, split them by purpose : demo or data. The filename will be
 the main_model name, suffixed by *_demo.xml* or *_data.xml*.
 
-For *controller*, the only file should be named *main.py*.
+For *controllers*, the only file should be named *main.py*. Otherwise, if you need to inherit an existing controller from another module, its name will be *<module_name>.py*. Unlike *models*, each controller should be contained in a separated file.
+
+For *static files*, since the resources can be used in different contexts (frontend, backend, both), they will be included in only one bundle. So, CSS/Less, JavaScript and XML files should be suffixed with the name of the bundle type. i.e.: *im_chat_common.css*, *im_chat_common.js* for 'assets_common' bundle, and *im_chat_backend.css*, *im_chat_backend.js* for 'assets_backend' bundle.
```

Comment:
- Regarding static files
- resources can be used in different contexts (..) and are included in only one bundle (-> use the present)
- remove the "So", you french pig http://franceisnice.com/wp-content/uploads/2015/04/tumblr_nkpelk2xIo1u0k6deo2_2501.gif